core/src/repair/quic_endpoint.rs

```diff
@@ -471,19 +500,27 @@ async fn send_requests_task(
     endpoint: Endpoint,
     connection: Connection,
     mut receiver: AsyncReceiver<LocalRequest>,
+    stats: Arc<RepairQuicStats>,
 ) {
     while let Some(request) = receiver.recv().await {
         tokio::task::spawn(send_request_task(
             endpoint.clone(),
             connection.clone(),
             request,
+            stats.clone(),
         ));
     }
 }
 
-async fn send_request_task(endpoint: Endpoint, connection: Connection, request: LocalRequest) {
+async fn send_request_task(
+    endpoint: Endpoint,
+    connection: Connection,
+    request: LocalRequest,
+    stats: Arc<RepairQuicStats>,
+) {
     if let Err(err) = send_request(endpoint, connection, request).await {
-        error!("send_request: {err:?}")
+        debug!("send_request: {err:?}");
```

Comment:
Can we output the destination address? That will be useful when examining the logs.

Reply:
done.